MergeSharpWebAPI/Controllers/TPTPGraphController.cs

```diff
@@ -124,13 +124,12 @@ public ActionResult<string> LookupNodes(int id)
 
         foreach (KeyValuePair<Guid, int> entry in IDMapping)
         {
-            Random rnd = new Random();;
+            Random rnd = new Random();
 
             var n = new Node(types[rnd.Next(0, types.Length)], entry.Value, $"{Math.Pow(-1, rnd.Next(1,3)) * rnd.Next(0, 200)} {Math.Pow(-1, rnd.Next(1, 3)) * rnd.Next(0, 200)}");
```

Comment:
not a change in this PR, but what is this code? is it just for testing? If so, please add a `TODO: <desired behaviour>`

Reply:
Truuu this was for picking random node positions bc we don't merge positions yet